scripts/deployer.ts

```diff
@@ -0,0 +1,280 @@
+import 'hardhat-deploy-ethers';
+import * as fs from 'fs';
+import { ContractReceipt, ContractTransaction } from 'ethers';
+import mainnetData from '../mainnet_deployed.json';
+import testnetData from '../testnet_deployed.json';
+import { getCurrentTimestamp } from 'hardhat/internal/hardhat-network/provider/utils/getCurrentTimestamp';
+import { ethers } from 'hardhat';
+import hre from 'hardhat';
+
+interface DeployedContract {
+  abi: Array<never>;
+  bytecode: string;
+  address: string;
+  block: number;
+  url: string;
+}
+
+interface DeployedData {
+  contracts: {
+    postageStamp: DeployedContract;
+    redistribution: DeployedContract;
+    staking: DeployedContract;
+    priceOracle: DeployedContract;
+    factory: DeployedContract;
+  };
+}
+
+// URL
+const mainnetURL = 'https://gnosisscan.io/address/';
+const testnetURL = 'https://goerli.etherscan.io/address/';
```

Comment:
can we add these into .env ?

Reply:
or fetch from hre config?

Reply:
I'd recommend putting this into the config.